### ISSUE TITLE
Missing Keys Resource Definition

### ISSUE THREAD

**Issue by mcelicalderon** [twilio/twilio-php#322]
*Note: These issues are for bugs and feature requests for the helper libraries. If you need help or support, please email help@twilio.com and one of our experts will assist you!*

**Version:** 5.0.0-RC6
**API Subdomain (api/taskrouter/ip_messaging):** API

There is no way to use `Keys` resource in version 5.0.0-RC6 of this library. This is the resource URL:

```
https://api.twilio.com/2010-04-01/Accounts/ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/Keys.json
```

**Comment by ihumanable** [comment]
Great catch, we will add this.

**Comment by mcelicalderon** [comment]
Awesome, thanks!

**Comment by jingming** [comment]
Released with `5.0.0-RC9`.
